node_modules/oae-tests/lib/util.js

```diff
@@ -256,18 +257,23 @@ var generateTestUsers = module.exports.generateTestUsers = function(restCtx, tot
                 return callback(err);
             }
 
-            _createdUsers.push({
-                'user': user,
-                'restContext': new RestContext(restCtx.host, {
-                    'hostHeader': restCtx.hostHeader,
-                    'username': username,
-                    'userPassword': 'password',
-                    'strictSSL': restCtx.strictSSL
-                })
-            });
+            // Manually verify the user their email address
+            PrincipalsDAO.setEmailAddress(user, email.toLowerCase(), null, function(err, user) {
```

Comment:
Is this actually necessary given we're setting the email in the `createUser` command?